src/server/polaris-context.ts

```diff
@@ -5,4 +5,5 @@ export interface PolarisContext extends PolarisBaseContext {
     headers: PolarisRequestHeaders;
     body: any;
     pubSub?: PubSub;
+    extensions: any;
```

Comment:
i think that you dont need an extension object inside polaris-context, but you need an irrelevantEntities object and that's it. Extensions are only for the response, just take it from the irrelevantEntities object inside the polaris-context.
In addition for that, please create a ts type for every object you create, and do not put **any**